src/utils/__tests__/fixtures/handlingTimesBySlug.json

```diff
@@ -0,0 +1,122 @@
+{
```

Comment:
This information is already in the category fixture and can be by adding a selector in `src/utils/__tests__/fixtures/index.js`. That way is no extra json file needed